danger-klaxit/spec/klaxit_spec.rb

```diff
@@ -398,7 +398,7 @@ def priv
       describe "#new_ruby_files_excluding_spec_rubocop_and_migrations" do
         before do
           allow(@plugin).to receive(:new_ruby_files_excluding_spec) do
-            %W(db/migrate/mimimi.rb #{Dir.pwd}/script/rm_rf_slash.rb app/good.rb)
+            %W(db/migrate/mimimi.rb #{Dir.pwd}/script/rm_rf_slash.rb app/workers/migrations/migration.rb app/good.rb)
```

Comment:
It's best with the right length;)

```suggestion
            %W(
              db/migrate/mimimi.rb #{Dir.pwd}/script/rm_rf_slash.rb 
              app/workers/migrations/migration.rb app/good.rb
            )
```